warplib/integer.cairo

```diff
@@ -13,3 +16,20 @@ fn get_u128_try_from_felt_result(value: felt252) -> u128 {
     return resp.unwrap();
 }
 
+/// Conversions.
+fn felt252_into_bool(val: felt252) -> bool {
+    let mut serialization_array: Array<felt252> = ArrayImpl::<felt252>::new();
+    ArrayImpl::<felt252>::append(ref serialization_array, val);
+    let mut span_serialization_array = ArrayImpl::<felt252>::span(@serialization_array);
+    let resp_option = BoolSerde::deserialize(ref span_serialization_array);
+    let resp = OptionTraitImpl::<bool>::unwrap(resp_option);
+    resp
+}
+
+fn bool_into_felt252(val: bool) -> felt252 {
+    let mut serialization_array: Array<felt252> = ArrayImpl::<felt252>::new();
+    BoolSerde::serialize(ref serialization_array, val);
+    let resp_option = ArrayImpl::pop_front(ref serialization_array);
+    let resp = OptionTraitImpl::<felt252>::unwrap(resp_option);
+    resp
+}
```

Comment:
Although I am in favor of serde, I think this might be an overkill?

Reply:
It'd be fine if we generalized that to a function `fn add_to_memory(val: Serde::<T>)` which would serialize any value and insert it into the memory

Reply:
WYT @rodrigo-pino ?

Reply:
I agree @piwonskp